index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>assignment01</title>
+    <script src="jquery-3.4.1.min.js"></script>
+<script src="script.js"></script>
+</head>
+<body>
+    <div id="test"></div>
```

Comment:
I would hope to see a block comment here that contains your name and the file name/assignment/instructor name. In future assignments, I will dock points for not including that block comment.